modules/ocf_puppet/manifests/puppetmaster.pp

```diff
@@ -3,9 +3,12 @@
     ['puppetserver', 'puppet-lint', 'augeas-tools']:;
   }
 
-  service { 'puppetserver':
-    enable  => true,
-    require => Package['puppetserver'],
+  # This defines Service['puppetserver'], so we can't do it ourselves.
+  class { 'puppetdb::master::config':
+    puppetdb_server => 'thunder.ocf.berkeley.edu',
```

Comment:
This could probably be removed now that the SAN cert is generated, as the default here is `puppetdb`, so I'll go change that.

Reply:
I was wrong about the default, it defaults to the local host, but I've changed this to a more general `'puppetdb'` and that's better in case the hostname needs to change.